### PR TITLE
Short input component

### LINKED ISSUE #63: `Input Short` Component
Screen Shot 2022-08-22 at 9 09 04 PM

Create the default, error, hover, active, disabled, and error with caption states for the `Input Short` Component

### PR DESCRIPTION
**Pull request by ajo01**
Preview of component
*added black background below to show text

Screen Shot 2022-08-28 at 7 57 53 PM

Closes #63

### CODE REVIEW

File: src/components/input/input-short/InputShort.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+const InputShort = ({ disabled, error, errorMessage }) => {
+  return (
+    <div>
+      <InputShortComponent
+        id="input-short-component"
+        disabled={disabled}
+        error={error}
+      />
+      {error && !disabled && <ErrorMessage>{errorMessage}</ErrorMessage>}
```
Comment (meleongg) [inline src/components/input/input-short/InputShort.js:39]: Will there eventually be logic to produce an error message if the text is too long for the short answer portion? Just wondering if that logic should be included here

**Review by meleongg — COMMENTED** [review]
I saw that the folder path was `/components/**input**/input-short`. In my PR, I did `/components/inputlonganswer`. Wondering which is the preferred format?

File: src/components/input/input-short/InputShort.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+`;
+
+const InputShort = ({ disabled, error, errorMessage }) => {
+  return (
+    <div>
+      <InputShortComponent
+        id="input-short-component"
+        disabled={disabled}
+        error={error}
+      />
```
Comment (DonaldKLee) [inline src/components/input/input-short/InputShort.js:38]: Are we adding placeholder text? If so, this is missing a placeholder attribute
image

**Review by DonaldKLee — CHANGES_REQUESTED** [review]

File: src/components/input/input-short/InputShort.js (hunk 2 above)
Comment (ajo01) [reply src/components/input/input-short/InputShort.js]: added a placeholder prop

File: src/components/input/input-short/InputShort.js (hunk 1 above)
Comment (ajo01) [reply src/components/input/input-short/InputShort.js]: I didn't initially add it as there wasn't a design for it, but I do think we need error handling for InputShort as well. I've added error handling if the text is too long

**Comment by ajo01** [comment]
> I saw that the folder path was `/components/**input**/input-short`. In my PR, I did `/components/inputlonganswer`. Wondering which is the preferred format?

It'll be best to group the components by their type. So group all buttons under a button folder and the long input and short input under an input folder. I'll make a separate PR for reorganizing the buttons later

File: src/components/input/input-short/InputShortCSS.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  gap: 10px;
+  color: white;
+  outline: none;
+  z-index: 0;
+
+  padding: 13px;
+  font-family: "HK Grotesk", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
```
Comment (meleongg) [inline src/components/input/input-short/InputShortCSS.js:19]: Small nit: should `font-size` be in `rem`?

Comment (meleongg) [reply src/components/input/input-short/InputShortCSS.js:19]: Otherwise, everything else lgtm

Comment (ajo01) [reply src/components/input/input-short/InputShortCSS.js]: made the changes! make sure to click Approve for the PR

**Review by meleongg — APPROVED** [review]
lgtm!

**Review by DonaldKLee — APPROVED** [review]
Lgtm! :)

**Pull request merged**